roles/rsyslog/defaults/main.yaml

```diff
@@ -38,21 +38,21 @@ rsyslog_capabilities: []
 # Enable or disable unprivileged ``rsyslogd`` operation. Warning, enabling this
 # option requires additional configuration outside of the ``rsyslog``
 # role.
-rsyslog_unprivileged: '{{ "True"
+rsyslog_unprivileged: '{{ "true"
                            if (ansible_distribution in [ "CentOS", "RedHat", "Fedora" ])
-                           else "False" }}'
+                           else "false" }}'
```

Comment:
can't you simplify to just `'{{ ansible_distribution in [ "CentOS", "RedHat", "Fedora" ] }}'`, given that this should be a boolean expression already?
Also, explicit tests for distribution should be discouraged, it would be better to use a distribution-specific var file, but let's keep it for later, this is a bigger change.

Reply:
oh and I think this has another problem: you probably want boolean values true/false and not strings "true"/"false", so the values should not be quoted.

Reply:
Opening another PR for "true"/"false" --> true/false...

Reply:
um, why? The code is not there anymore.

Reply:
@pcahyna 
> um, why? The code is not there anymore.

Because I fixed it as you suggested and merged it with --force...

Reply:
@nhosoi I don't understand. Here: https://github.com/linux-system-roles/logging/blob/f63b8439e2faa4586d3855c95446def98e39f586/roles/rsyslog/defaults/main.yaml#L42 
there is no trace of "true" ad "false" anymore because you fixed it to not need it at all. (My comment was about the original code and did not apply to the fixed version.) Therefore, you don't need to open a new PR for "true"/"false" --> true/false, unless they appear in some other places in the code.

Reply:
Ahhh, I see...  I somehow misunderstood you were talking about "true" and "false" in general, as I thought I made the mistake many places, but it looks I didn't [1].  *phew*  Sorry for my confusion and noise.
[1]
```
$ find logging -type f | xargs egrep "\"true\"|\"false\""
logging/roles/rsyslog/roles/README.md:  - If `rsyslog_default` equals to "true", It is required to set for all logs you don't want to be processed by the default rules:
logging/roles/rsyslog/roles/README.md:    set $.send_targets_only = "true";
logging/roles/rsyslog/roles/input_roles/ovirt/defaults/main.yaml:              set $.send_targets_only = "true";
logging/roles/rsyslog/roles/input_roles/ovirt/defaults/main.yaml:              set $.send_targets_only = "true";
```